scripts/helpers.sh

```diff
@@ -33,9 +33,23 @@ battery_status() {
 	elif command_exists "upower"; then
 		# sort order: attached, charging, discharging
 		for battery in $(upower -e | grep battery); do
-			upower -i $battery | grep state | awk '{print $2}'
+			upower -i "$battery" | grep state | awk '{print $2}'
 		done | sort | head -1
 	elif command_exists "acpi"; then
 		acpi -b | grep -oi 'discharging' | awk '{print tolower($0)}'
 	fi
 }
+
+ac_online() {
+    local toggle_ac_status="$(get_tmux_option "@batt_toggle_ac_status" "false")"
+    [[ "${toggle_ac_status}" == "false" ]] && return 1
+
+    # TODO: what about pmset and acpi?
+    if command_exists "upower"; then
+        [[ "$(upower -i /org/freedesktop/UPower/devices/line_power_AC | awk -F: '/online/ {gsub(/ /, "", $NF); print $NF}')" == "yes" ]]
+    elif [[ -e /sys/class/power_supply/AC/online ]]; then
+        [[ "$(cat /sys/class/power_supply/AC/online)" -eq 1 ]]
```

Comment:
What is the reason to use `-eq` here, while you changed those to `==` in the code above?